test_runner/fixtures/neon_fixtures.py

```diff
@@ -2440,7 +2442,7 @@ def wait_for_upload(pageserver_http_client: NeonPageserverHttpClient,
                     timeline: uuid.UUID,
                     lsn: int):
     """waits for local timeline upload up to specified lsn"""
-    for i in range(10):
+    for i in range(20):
```

Comment:
How do we make sure that `20` is enough but not other bigger numbers? Did you test this change in another branch?

Reply:
I checked it locally and it didnt fail several times, it didnt've failed with old value either... I'll rerun the job here several times to make sure that this is enough. 

I think here we hit some temporary slowdown with s3 maybe because of the limits. So Incrementing it to twice the value should be enough